config/puma.rb

```diff
@@ -145,7 +145,7 @@
 # is preserved across a phased-restart. (incompatible with preload_app)
 # (off by default)
 
-prune_bundler
+# prune_bundler
```

Comment:
`prune_bundler` is required to phase restart puma. However, it's only enabled on `test` and `development` envs, where we never phase restart.

It's causing us a lot of problems to run our test suite in Puma 6, so I'm disabling it.

Reply:
So, this `config/puma.rb` file is only used for dev/test?

What happens if `preload_app!` is enabled also? 

If it can be, we could probably simplify the config, and remove `openshift/config/puma.rb` ?